advanced-coroutines-codelab/finished_code/src/main/java/com/example/android/advancedcoroutines/PlantListViewModel.kt

```diff
@@ -141,10 +133,10 @@ class PlantListViewModel internal constructor(
      */
     fun clearGrowZoneNumber() {
         growZone.value = NoGrowZone
-        growZoneChannel.offer(NoGrowZone)
+        growZoneFlow.value = NoGrowZone
 
         // initial code version, remove during flow rewrite
-        launchDataLoad { plantRepository.tryUpdateRecentPlantsCache() }
+        // launchDataLoad { plantRepository.tryUpdateRecentPlantsCache() }
```

Comment:
same here. this is the `finished_code` module, either this should be uncommented or removed